app/api/delete-event/route.ts

```diff
@@ -0,0 +1,50 @@
+import { getAppServerSession, isAdmin } from "@/app/auth"; // ensuring that only trusted users can access
+import { NextResponse } from "next/server"; // structuring response to users
+import { prisma } from "../db"; // bridge between me and db
+
+/*
+    1. authenticate user: allowDelete
+    2. delete the event from db: deleteEvent
+    3. reflect change to user interface: DELETE
+*/
```

Comment:
Maybe instead of comments at the top of the file, these should be separated into doc comments for individual functions

Reply:
or can I just get rid of the comments since the team members won't need them? I had them just for me to have a better understanding of what the functions did when I was writing them.